### PR TITLE
Companion function depreciation.

### PR DESCRIPTION
**Pull request by Wutname1**
Not sure how you want to handle these ones.

These calls have only supported mounts since MOP some do not return/do anything. There are other API calls to do mount specific stuff so I'm guessing these will disappear fully at some point.

### CODE REVIEW

**Comment by Ketho** [comment]
Oh, those are marked on the [page](https://wowpedia.fandom.com/wiki/API_CallCompanion) itself with a template like `{{protectedhwapi|type=deprecated}}` so I suppose there is no need to also put that in the description.

File: conflicts.lua [hunk 1]
```diff
@@ -1925,29 +1925,36 @@ local t = {
...
-	["InRepairMode"] = [=[Lets you know if your cursor is in repair mode. When your cursor is in repair mode, you can click on equipped items as well as items in your inventory to repair them.]=],
+	-- ["InRepairMode"] = [=[Returns true if your cursor is in repair mode.]=],
+	-- ["InRepairMode"] = [=[Lets you know if your cursor is in repair mode. When your cursor is in repair mode, you can click on equipped items as well as items in your inventory to repair them.]=],
+	["InRepairMode"] = [=[True if cursor is in repair mode.]=],
 
-	["InboxItemCanDelete"] = [=[Boolean check for whether a message is returnable to the sender.]=],
-	["InboxItemCanDelete"] = [=[Boolean function for determining whether a message is returnable.]=],
+	-- ["InboxItemCanDelete"] = [=[Boolean check for whether a message is returnable to the sender.]=],
+	-- ["InboxItemCanDelete"] = [=[Boolean function for determining whether a message is returnable.]=],
+	["InboxItemCanDelete"] = [=[False if a mailed item or money is returnable, true otherwise.]=],
```
Comment (Wutname1) [inline conflicts.lua]: Not sure about this one. I thought the Return description explained what it does better than the other 2 https://wowpedia.fandom.com/wiki/API_InboxItemCanDelete Figured toss it in here let you make the final call

Comment (Ketho) [reply conflicts.lua]: yea I agree with the return description. I'll still word it slightly different though

**Comment by Ketho** [comment]
I do kind of want the actual function to be verified. Not by just looking at the description but by actually testing the in-game behavior or looking at the arguments/returns and being more clear (when possible)

For example `["HasPetUI"] = [=[True if the player has a pet interface.]=],` does not really tell me what "having a pet interface" means. I had to go in-game to see it return true when summoning a hunter/warlock pet.

**Comment by Wutname1** [comment]
> I had to go in-game to see it return true when summoning a hunter/warlock pet.

I did not catch that one when I logged into my hunter I had the pet out already so it returned true whereas my druid said false. 

I have been trying to confirm most of them. Especially ones that say "Returns 1" my first thought is always "Does it really?" Most have been changed to bool and not number returns.

**Pull request merged**